src/components/Review.jsx

```diff
@@ -0,0 +1,34 @@
+import { useParams } from "react-router";
+import * as api from "../api.js";
+import { useEffect, useState } from "react";
+
+function Review() {
+  const { review_id } = useParams();
+  const [review, setReview] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    api.fetchReviewById(review_id).then((review) => {
+      setReview(review);
+      setIsLoading(false);
+    });
+  }, [review_id]);
+
+  return isLoading ? (
+    <p>is loading...</p>
+  ) : (
+    <div className="Review">
+      <h2>{review.title}</h2>
+      <p>{review.review_body}</p>
+      <p>{review.category}</p>
+      <p>{review.designer}</p>
+      <p>{review.owner}</p>
+      <p>{review.created_at}</p>
```

Comment:
this date isn't formatted and isn't very user friendly, either manipulate it or install a package to do it for you (I like moment.js)